app/client/components/Upload.tsx

```diff
@@ -154,7 +155,12 @@ export class Upload extends React.Component<UploadProps, UploadState> {
             // TODO: stuff the link in the user profile
             const hash: string = xorName.toString("base64");
             console.log(`uploaded video to: frames://${hash}`);
```

Comment:
I left this one in because it is currently the only way to learn the frames URL that the video was uploaded to. Obviously we need to figure out a way to get the FramesURLBar to be responsive to location changes and display always display the right URL.